manifests/repo/nodesource.pp

```diff
@@ -9,32 +9,26 @@
   $proxy_username = $nodejs::repo_proxy_username
   $url_suffix     = $nodejs::repo_url_suffix
 
-  case $::osfamily {
+  case $facts['os']['family'] {
     'RedHat': {
-      if $::operatingsystemrelease =~ /^5\.(\d+)/ {
-        include ::epel
-        $dist_version  = '5'
-        $name_string   = 'Enterprise Linux 5'
-      }
-
-      elsif $::operatingsystemrelease =~ /^6\.(\d+)/ {
+      if $facts['os']['release']['full'] =~ /^6\.(\d+)/ {
```

Comment:
The ```major``` should be there in facts too and doesn't need a regex. You can even simplify the later code to use those facts in ```$dist_version``` and ```$name_string```.

Reply:
Done